packages/did-core-test-server/suites/did-spec/did-core-properties.js

```diff
@@ -0,0 +1,31 @@
+const utils = require('./utils');
+const {isValidDID} = utils;
+
+const jsonMediaTypes = ['application/did+ld+json', 'application/did+json'];
+
+const generateDidCorePropertiesTests = ({did, resolutionResult}) => {
+  const {didDocument} = resolutionResult;
+
+  it('5.1.1 DID Subject - The value of id MUST be a string that ' +
```

Comment:
@OR13 -- Thinking of prefacing tests with the section they're in to make it easier to map these sorts of things to sections of the specification. Thoughts?

Reply:
yeah that sounds good. I wrote this respec to jest code gen here: https://github.com/transmute-industries/did-core/tree/main/packages/did-core-tests/__generated__

which automatically handled all this stuff, but it harder to do that for tests related to did methods vs the spec itself.